lib/typed-events.ts

```diff
@@ -241,3 +301,11 @@ export type DecorateAcknowledgementsWithMultipleResponses<E> = {
     ? (...args: ExpectMultipleResponses<Params>) => Result
     : E[K];
 };
+
+export type RemoveAcknowledgements<E> = {
```

Comment:
I'm not sure to understand here, shouldn't the events with an acknowledgement be flltered out?